mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -702,7 +702,7 @@
 
     <!-- History -->
     <!-- Text for the button to search all history -->
-    <string name="history_search">Enter search term</string>
+    <string name="history_search">Enter search terms</string>
```

Comment:
@rocketsroger: please update the string ID so that localizers get notified of this change in Pontoon